addons/portal_rating/static/src/scss/portal_rating.scss

```diff
@@ -56,22 +56,11 @@ $o-w-rating-star-color: #FACC2E;
 /* Star Widget */
 .o-mail-Composer-starCard{
     width: max-content;
-    margin-bottom: 5px;
     .o-mail-Composer-stars {
         display: inline-flex;
         color: #FACC2E;
         margin-right: 15px;
     }
-
-    .o-mail-Composer-stars i {
-        padding-right: 1px;
-        padding-left: 1px;
-        text-align: center;
-    }
-
-    .o-mail-Composer-stars.enabled{
-        cursor: pointer;
-    }
```

Comment:
Same comment for the rest of the style, you have to apply it as well in `portal_tools.xml`